src/runtime_src/core/common/memaccess.cpp

```diff
@@ -0,0 +1,217 @@
+/**
+ * Copyright (C) 2016-2021 Xilinx, Inc
+ * Copyright (C) 2022 Advanced Micro Devices, Inc. - All rights reserved
+ * 
+ * Author: Sonal Santan
+ * Simple command line utility to interact with PCIe devices
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+#define XRT_CORE_COMMON_SOURCE
+#include "memaccess.h"
+
+#include <iostream>
```

Comment:
Please sort the include files.

Reply:
They are sorted with the exception of the unistd include. So I moved that up. Was that the line you meant?